projects/storefrontstyles/scss/components/configurator/_config-attribute-header.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../theme';
+
+%cx-config-attribute-header {
+  .cx-config-attribute-label {
+    padding-top: 10px;
+    padding-right: 6px;
+    font-size: $font-size-base;
```

Comment:
Sorry, I didn't see this one before.
We are not importing sass theme variables into components anymore, they are only to be used in the `cxbase` which we don't touch that much anymore. 
There's no need to declare the base font but if you must then please use instead `@include type()` or `@include type("5")` if you need it bold. Accepted values are 1 to 8 and none.
You can check this file as an example for that matter `components/checkout/steps/_progress.scss`

Reply:
Done